sample/src/main/res/values/colors.xml

```diff
@@ -9,13 +9,17 @@
     <color name="blueThemeGradientStartColor">#62cdf7</color>
     <color name="blueThemeGradientEndColor">#515efe</color>
     <color name="blueThemeItemTextColor">#5369fd</color>
+    <color name="blueThemeAlertDialogButtonTextColor">#5260fe</color>
 
     <color name="redThemeGradientStartColor">#ff8139</color>
     <color name="redThemeGradientEndColor">#fa2d64</color>
     <color name="redThemeItemTextColor">#fa3062</color>
+    <color name="redThemeAlertDialogButtonTextColor">#fd4049</color>
 
     <color name="itemPriceTextColor">#878787</color>
     <color name="grayBackgroundColor">#f9fafb</color>
     <color name="footerBackgroundColor">#c1c1c1</color>
 
+    <color name="black_54">#8a000000</color>
```

Comment:
What does "54" mean?

Reply:
It means black with alpha `54%`. I'll renamed it to `black_a54`